api/src/main/java/org/openmrs/api/db/DelegatingFullTextSession.java

```diff
@@ -56,82 +60,83 @@ public FullTextQuery createFullTextQuery(Query luceneQuery, Class<?>... entities
 		if (entities.length > 1) {
 			throw new DAOException("Can't create FullTextQuery for multiple persistent classes");
 		}
-		
-		if (log.isDebugEnabled()) {
-			log.debug("Creating new FullTextQuery instance");
-		}
-		
+
+		log.debug("Creating new FullTextQuery instance");
+
 		Class<?> entityClass = entities[0];
 		FullTextQuery query = delegate.createFullTextQuery(luceneQuery, entityClass);
-		
-		if (log.isDebugEnabled()) {
-			log.debug("Notifying FullTextQueryCreated listeners...");
-		}
-		
-		//Notify listeners, note that we intentionally don't catch any exception from a listener
-		//so that failure should just halt the entire creation operation, this is possible because 
-		//the default ApplicationEventMulticaster in spring fires events serially in the same thread
-		//but has the downside of where a rogue listener can block the entire application.
```

Comment:
the log `log.debug("Notifying {} listeners", entityClass.getName());` was before the comment. please move it back.